### ISSUE TITLE
Manage heading with require-description-complete-sentence

### ISSUE THREAD

**Issue by PierBJX** [gajus/eslint-plugin-jsdoc#1220]
## Expected behavior

I expect to not need to add a dot at the end of a heading in a JS description.

## Actual behavior

In our coding rules, it is needed to add a description with some headings (only for class). However, there is a heading which does not contain any sentence but only code sample. Hence, it raises an error because after the heading `# Example` it expects a dot.

## ESLint Config

```js
    "jsdoc/require-description-complete-sentence": [
      "error",
      {
        "abbreviations": [
          "etc",
          "e.g.",
          "i.e."
        ]
      }
    ],
```

## ESLint sample

```js
/**
 * ### Overview
 * My class is doing.
 *
 * ### Example
 * ```javascript
 *  const toto = 'toto';
 * ```
 */
export class ClassExemple {
}
```

## Environment

- Node version:  v18.18.2
- ESLint version  v8.56.0
- `eslint-plugin-jsdoc` version: 46.10.1

**Comment by PierBJX** [comment]
It seems that this fix introduces another issue. Indeed, now it does not raise an error if there is no dot at the end of the sentence below the heading. Here there is no more issue with that

```
/**
 * ### Overview
 * My class is doing
 */
```

**Comment by brettz9** [comment]
Sorry about the naive fix... Expect it should be fixed now...

**Comment by PierBJX** [comment]
Yes perfect ! thanks
